.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules
 .tempChromeProfileForDebug
-*.map 
\ No newline at end of file
+*.map 
```

Comment:
Could those updates be moved in your specific gitIgnore ?

Reply:
Will do